Misc/python.man

```diff
@@ -431,6 +431,10 @@ values.
 
 The integer must be a decimal number in the range [0,4294967295].  Specifying
 the value 0 will disable hash randomization.
+.IP PYTHONHISTORY
+If this is set you can change the location of a
+python_history file, by default it will be ~/.python_history.
```

Comment:
File names should be italic in man pages so `\fI~/.python_history\fI` or

```man
python_history file, by default it will be
.IR ~/.python_history .
```

You may need to escape ~ or / characters.